lib/tbot/config/config.go

```diff
@@ -186,6 +186,11 @@ type BotConfig struct {
 	// such as tctl or the Kubernetes operator.
 	AuthServerAddressMode AuthServerAddressMode `yaml:"-"`
 
+	// JoiningURI is a joining URI, used to supply connection and authentication
+	// parameters in a single bundle. If set, the value is parsed and merged on
+	// top of the existing configuration during `CheckAndSetDefaults()`.
+	JoiningURI string `yaml:"uri"`
```

Comment:
I've been debating the field name a bit:
- `uri`: short and sweet
- `join_uri` / `joining_uri`: a bit more specific, maybe less confusing if we ever have more than one URI in the config file?

Reply:
Fair enough, renamed to `join_uri`!